apache/superset
# chore: Moves the stylesheets folder to the assets folder

### SUMMARY
Moves the `stylesheets` folder to the `assets` folder.

This work is part of [SIP-61](https://github.com/apache/superset/issues/13632)

@jinghua-qa @junlincc 

### TESTING INSTRUCTIONS
1 - Navigate through the application
2 - Check that all styles are loaded correctly

### ADDITIONAL INFORMATION
